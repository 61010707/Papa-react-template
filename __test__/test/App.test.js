import React from "react";
import { shallow } from "enzyme";
import App from "../../src/components/App.js";

describe("App", () => {
  const app = shallow(<App />);

  it("renders the title", () => {
    expect(app.find("h1").exists()).toBe(true);
  });

  it("should render correctly with no props", () => {
    const component = shallow(<App />);

    expect(component).toMatchSnapshot();
  });

});
