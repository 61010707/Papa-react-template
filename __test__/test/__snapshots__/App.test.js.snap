// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render correctly with no props 1`] = `ShallowWrapper {}`;
